Fishing Game/ViewControllers/FishingReel.swift

```diff
@@ -116,7 +116,7 @@ class FishingReel {
     // Uses one bait when a fish bites
     func useBait() {
         print("Used bait")
-        guard fishingDay?.tacklebox.baitCount != 0 else { return }
+        guard fishingDay?.tacklebox.baitCount ?? 0 <= 0 else { return }
```

Comment:
shouldn't this be greater than 0? if you have 0 bait or less you can't use the bait you don't have. . .